modules/sdk-coin-avaxc/src/avaxc.ts

```diff
@@ -6,184 +6,49 @@ import { bip32 } from '@bitgo/utxo-lib';
 import Keccak from 'keccak';
 import * as secp256k1 from 'secp256k1';
 import * as _ from 'lodash';
-import { BaseCoin as StaticsBaseCoin, CoinFamily, coins } from '@bitgo/statics';
+import { BaseCoin as StaticsBaseCoin, CoinFamily, coins, ethGasConfigs } from '@bitgo/statics';
 import {
   BaseCoin,
   BitGoBase,
   common,
   FeeEstimateOptions,
-  FullySignedTransaction,
-  HalfSignedAccountTransaction,
   InvalidAddressError,
   IWallet,
   KeyPair,
-  MethodNotImplementedError,
   ParsedTransaction,
   ParseTransactionOptions,
-  PresignTransactionOptions as BasePresignTransactionOptions,
-  SignTransactionOptions as BaseSignTransactionOptions,
+  Recipient,
   TransactionExplanation,
-  TransactionFee,
-  TransactionParams,
-  TransactionPrebuild as BaseTransactionPrebuild,
-  TransactionRecipient,
+  Util,
   VerifyAddressOptions,
-  VerifyTransactionOptions,
-  Wallet,
 } from '@bitgo/sdk-core';
-import { optionalDeps, TransactionBuilder as EthTransactionBuilder } from '@bitgo/sdk-coin-eth';
+import {
+  GetSendMethodArgsOptions,
+  optionalDeps,
+  RecoverOptions,
+  RecoveryInfo,
+  SendMethodArgs,
+  TransactionBuilder as EthTransactionBuilder,
+  TransactionPrebuild,
+} from '@bitgo/sdk-coin-eth';
 import { isValidEthAddress } from './lib/utils';
 import { KeyPair as AvaxcKeyPair, TransactionBuilder } from './lib';
-
-// For precreateBitgo
-interface PrecreateBitGoOptions {
-  enterprise?: string;
-  newFeeAddress?: string;
-}
-
-// For explainTransaction
-export interface ExplainTransactionOptions {
-  txHex?: string;
-  halfSigned?: {
-    txHex: string;
-  };
-  feeInfo: TransactionFee;
-}
-
-interface AvaxcTransactionParams extends TransactionParams {
-  gasPrice?: number;
-  gasLimit?: number;
-  hopParams?: HopParams;
-  hop?: boolean;
-}
-
-interface VerifyAvaxcTransactionOptions extends VerifyTransactionOptions {
-  txPrebuild: TransactionPrebuild;
-  txParams: AvaxcTransactionParams;
-}
-
-// For preSign
-interface PresignTransactionOptions extends TransactionPrebuild, BasePresignTransactionOptions {
-  wallet: Wallet;
-}
-
-interface EIP1559 {
-  maxPriorityFeePerGas: number;
-  maxFeePerGas: number;
-}
-
-export interface TransactionPrebuild extends BaseTransactionPrebuild {
```

Comment:
BREAKING CHANGE. No longer exported from package.

Reply:
how should I handle this?

Reply:
Needs to be documented in the footer of the commit message. https://www.conventionalcommits.org/en/v1.0.0/
```
feat: text

BREAKING CHANGE: description
BG-<number>
```

Reply:
Needs to be in commit footer.